File: java/metodos/funcion_para_contar_contactos.java
package metodos;
public class funcion_para_contar_contactos {
    public static int contarContactos(String[] contactos) {
        int contador = 0;
        for (int i = 0; i < contactos.length; i++) {
            if (contactos[i] != null && !contactos[i].trim().isEmpty()) {
                contador++;
            }
        }
        return contador;
    }
    public static void main(String[] args) {
        String[] contactos = {"Juan", "Maria", "", null, "Carlos", "Ana"};
        int total = contarContactos(contactos);
        System.out.println("Total de contactos registrados: " + total);
    }
}

File: visual_basic/metodos/funcion_para_contar_contactos/funcion_para_contar_contactos.vb
Imports System
Module funcion_para_contar_contactos
    Sub Main(args As String())
        Dim contactos As New List(Of String) From {
            "Juan", "Maria", "Carlos", "Ana"
        }
        Dim total As Integer = ContarContactos(contactos)
        Console.WriteLine("Total de contactos: " & total)
        Console.ReadKey()
    End Sub
    Function ContarContactos(ByVal lista As List(Of String)) As Integer
        Return lista.Count
    End Function
End Module